tmol/tests/score/disulfide/test_disulfide_energy_term.py

```diff
@@ -43,115 +44,62 @@ def test_annotate_disulfide_conns(
-    torch.arange(100, device=torch_device)
-
-    numpy.testing.assert_allclose(
-        gold_vals, scores.cpu().detach().numpy(), atol=1e-5, rtol=1e-5
-    )
-
-
-def test_whole_pose_scoring_module_jagged(
-    rts_disulfide_res, default_database, torch_device: torch.device
-):
-    rts_disulfide_60 = rts_disulfide_res[:60]
-    rts_disulfide_33 = rts_disulfide_res[:33]
-    p1 = PoseStackBuilder.one_structure_from_polymeric_residues(
-        default_database.chemical, res=rts_disulfide_res, device=torch_device
-    )
-    p2 = PoseStackBuilder.one_structure_from_polymeric_residues(
-        default_database.chemical, res=rts_disulfide_60, device=torch_device
-    )
-    p3 = PoseStackBuilder.one_structure_from_polymeric_residues(
-        default_database.chemical, res=rts_disulfide_33, device=torch_device
-    )
-    pn = PoseStackBuilder.from_poses([p1, p2, p3], device=torch_device)
-
-    disulfide_energy = DisulfideEnergyTerm(
-        param_db=default_database, device=torch_device
-    )
-    for bt in pn.packed_block_types.active_block_types:
-        disulfide_energy.setup_block_type(bt)
-    disulfide_energy.setup_packed_block_types(pn.packed_block_types)
-    disulfide_energy.setup_poses(pn)
-
-    disulfide_pose_scorer = disulfide_energy.render_whole_pose_scoring_module(pn)
-    scores = disulfide_pose_scorer(pn.coords)
-
-    gold_scores = numpy.array([[-3.25716, -3.25716, -1.686726]], dtype=numpy.float32)
-    numpy.testing.assert_allclose(gold_scores, scores.cpu().numpy(), rtol=1e-5)
+class TestDisulfideEnergyTerm(EnergyTermTestBase):
+    energy_term_class = DisulfideEnergyTerm
+
+    @classmethod
+    def test_whole_pose_scoring_10(
+        cls, rts_disulfide_res, default_database, torch_device, update_baseline=False
+    ):
+        return super().test_whole_pose_scoring_10(
+            rts_disulfide_res, default_database, torch_device, update_baseline
+        )
+
+    @classmethod
+    def test_whole_pose_scoring_jagged(
+        cls,
+        rts_disulfide_res,
+        default_database,
+        torch_device: torch.device,
+        update_baseline=False,
+    ):
+        return super().test_whole_pose_scoring_jagged(
+            rts_disulfide_res, default_database, torch_device, update_baseline
+        )
+
+    @classmethod
+    def test_whole_pose_scoring_gradcheck(
+        cls, rts_disulfide_res, default_database, torch_device
+    ):
+        return super().test_whole_pose_scoring_gradcheck(
+            rts_disulfide_res,
+            default_database,
+            torch_device,
+            eps=1e-3,
+            atol=1e-2,
+            rtol=5e-3,
+        )
+
+    @classmethod
+    def test_block_scoring(
+        cls, rts_disulfide_res, default_database, torch_device, update_baseline=False
+    ):
+        rts_res = rts_disulfide_res[2:4] + rts_disulfide_res[21:23]
+        return super().test_block_scoring(
+            rts_res, default_database, torch_device, update_baseline
+        )
```

Comment:
@aleaverfay can you advise on how to accomplish discontinuous residues with the new pose_stack_from_pdb? Should I just make this go from 2:23 instead?